cabal.project

```diff
@@ -22,18 +58,62 @@ source-repository-package
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-shell
-  tag: 4be5660c721cae4b001187ec614ef5cf4fad8fc7
+  tag: f6fcda280ddd1d4ef87e5203d65d132f924a6e39
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/cardano-sl-x509
+  tag: e8bfc1294e088f90e5ae0b4aedbc82ee46ac5ee4
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/iohk-monitoring-framework
+  subdir: iohk-monitoring
+  tag: 9129241a1306e2418bb775c63ad8a3ccd785c068
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/iohk-monitoring-framework
+  subdir:   contra-tracer
+  tag: 9129241a1306e2418bb775c63ad8a3ccd785c068
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: b3a46e40ba95204866890caf9713c5c2af282141
+  tag: 1c291da70decc003b474b192e335e86e80c252b3
   subdir: ouroboros-network
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/ouroboros-network
-  tag: e40e48f9aa86bab9c2a4b43b65747de030069b84
+  tag: 1c291da70decc003b474b192e335e86e80c252b3
   subdir: ouroboros-consensus
 
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/ouroboros-network
+  tag: 1c291da70decc003b474b192e335e86e80c252b3
+  subdir: typed-protocols
+
+source-repository-package
+  type: git
+  location: https://github.com/input-output-hk/ouroboros-network
+  tag: 1c291da70decc003b474b192e335e86e80c252b3
+  subdir: io-sim-classes
+
+source-repository-package
+  type: git
+  location: https://github.com/well-typed/cborg
+  tag: 80fbe0ee5e67a5622e2cb9eaa9d8594a2214322d
+  subdir: cborg
+
+source-repository-package
+  type: git
+  location: https://github.com/joelwilliamson/bimap
+  tag: 997fbb38b08dec14d225d064dac05b0a85f4ceae
 
+constraints:
+  ip < 1.5,
+  ekg-prometheus-adapter >= 0.1.0.4,
+  hedgehog >= 1.0,
+  libsystemd-journal >= 1.4.4
```

Comment:
Someone's text editor should be configured to always make sure the file ends in a newline.